src/components/CurrencyInputPanel/index.tsx

```diff
@@ -49,60 +50,65 @@ interface CurrencyInputPanelProps {
   showCommonBases?: boolean
 }
 
-const CurrencyInputPanel:React.FC<CurrencyInputPanelProps> = ({
-  value,
-  onUserInput,
-  onMax,
-  showMaxButton,
-  label,
-  currency,
-  remainingSupply = '',
-  hideBalance = false,
-  hideInput = false,
-  id,
-  showCommonBases }) => {
-    const { account } = useWeb3React()
+const CurrencyInputPanel: React.FC<CurrencyInputPanelProps> = (
+  {
+    value,
+    onUserInput,
+    onMax,
+    showMaxButton,
+    label,
+    currency,
+    remainingSupply = '',
+    hideBalance = false,
+    disabled,
+    hideInput = false,
+    id,
+    showCommonBases,
+  }) => {
+  const { account } = useWeb3React()
 
-    return (
-      <InputPanel>
-        <Text fontSize="14px" margin='5px 0px'>
-          {label}
-        </Text>
-        <Container>
-          <InputRow>
+  return (
+    <InputPanel>
+      <Text fontSize='1em' margin='5px 0px'>
```

Comment:
May comment ako here kuys, using em, pt and px is same effect. Yung update ko sa timer fix, I used vh, vw. mas responsive yun.